packages/components/src/internal/components/entities/constants.ts

```diff
@@ -62,8 +64,38 @@ export const AssayResultDataType: EntityDataType = {
     uniqueFieldKey: 'RowId',
     dependencyText: undefined,
     filterCardHeaderClass: 'filter-card__header-purple',
+    sampleFinderCardType: 'assaydata',
 };
 
+export const SamplePropertyDataType: EntityDataType = {
+    allowSingleParentTypeFilter: true,
+    allowRelativeDateFilter: true,
```

Comment:
Is there a reason relative date filtering doesn't work for other data types? It seems a generally useful thing, but if it entails more work, we can add that in in a different story.

Reply:
The relative filtering was added to support built-in reports with those filters. I kinda just made it 'work/not broken', without UX consideration. If proven useful, we can handle the broader roll out in a different story.